src/src/radio.c

```diff
@@ -1396,6 +1396,13 @@ uint8_t otPlatRadioGetCslAccuracy(otInstance *aInstance)
 }
 #endif // OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
 
+uint8_t otPlatRadioGetCslClockUncertainty(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    return 0;
```

Comment:
Perfect accuracy is not achieved yet. I can observe Child Update Response being retransmitted 100 % of the times. This means receive window is too small for short elapsed time. When elapsed time increases, clock accuracy makes the receive window larger and it works better.

I observe better results for short elapsed times with a value of `20` (200 us) for `otPlatRadioGetCslClockUncertainty`.

Reply:
Fixed.